src/components/content-editor/resource-editor.tsx

```diff
@@ -141,13 +141,15 @@ class ResourceEditor extends React.Component<Props, State> {
   private renderResourceField = () => {
     const { classes, accessToken, resource } = this.props;
 
+    console.log("static resource - type: ", resource.type);
+
```

Comment:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAARGH!
```suggestion
    console.log("static resource - type: ", resource.type);

```